competitions/LLIIA2020/etl/deploy

```diff
@@ -124,7 +142,6 @@ if ! echo "831bd76551d1d21aea16820aa64463d27d3001a1  $DATA_DIR/$PROPOSALS_CSV" |
   exit 1
 fi
 
-
```

Comment:
Spurious line removal (again, no big deal).

For other things that are the same as in the previously reviewed file, I'm not mentioning them twice, FYI.